### PR TITLE
Added forceCustomization option for vCloud Director

### PR DESCRIPTION
**Pull request by juanfont**
### Description

These commits add support for forceCustomization param when deploying a node in vCloud Director.

See https://www.vmware.com/support/vcd/doc/rest-api-doc-1.5-html/types/DeployVAppParamsType.html.
### Status

Ready for review.
### Checklist (tick everything that applies)
- [X] [Code linting](http://libcloud.readthedocs.org/en/latest/development.html#code-style-guide) (required, can be done after the PR checks)
- [X] Documentation
- [X] [Tests](http://libcloud.readthedocs.org/en/latest/testing.html)
- [ ] [ICLA](http://libcloud.readthedocs.org/en/latest/development.html#contributing-bigger-changes) (required for bigger changes)

### CODE REVIEW

**Comment by tonybaloney** [comment]
Great contribution, thanks @juanfont 👍

File: libcloud/compute/drivers/vcloud.py [hunk 1]
```diff
@@ -1044,8 +1063,6 @@ def ex_deploy_node(self, node):
                                       method='POST',
                                       headers=headers)
         self._wait_for_task_completion(res.object.get('href'))
-        res = self.connection.request(get_url_path(node.id))
```
Comment (Kami) [inline libcloud/compute/drivers/vcloud.py:1047]: Is it intentional and desired that this function doesn't return anything now?

Or did I miss something?

Comment (juanfont) [inline libcloud/compute/drivers/vcloud.py:1047]: It's intentional. That's an internal method that handles both vapp's and vm's. It's meant to be called only from ex_deploy_node (where the node is indeed returned).

Comment (Kami) [inline libcloud/compute/drivers/vcloud.py:1047]: Ah, sorry, I was confused by the diff (I didn't fully expand it so I thought it's part of the public method).

**Comment by Kami** [comment]
Merged, thanks!

**Pull request merged**